dialogue-clients/src/test/java/com/palantir/dialogue/clients/DialogueClientsDnsIntegrationTest.java

```diff
@@ -190,7 +193,7 @@ void dns_refresh_updates_in_flight_requests_reloading_channel() throws UnknownHo
         dnsEntries.put(hostOne, InetAddress.getByAddress(hostOne, new byte[] {127, 0, 0, 2}));
 
         TaggedMetricRegistry metrics = new DefaultTaggedMetricRegistry();
-        Counter activeTasks = ClientDnsMetrics.of(metrics).tasks(DnsPollingSpec.RELOADING_FACTORY.kind());
+        Counter activeTasks = ClientDnsMetrics.of(metrics).tasks(ChannelNames.reloading("foo"));
```

Comment:
do we want to update these tests to us a higher retry interval + Awaitility instead? Could save that for a separate PR also.